packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -35,12 +35,21 @@ const CFDsListing = () => {
 
     const accounts_sub_text = is_eu ? localize('Account Information') : localize('Compare accounts');
 
-    const getShortCode = (account: TDetailsOfEachMT5Loginid) => {
-        return account.landing_company_short &&
-            account.landing_company_short !== 'svg' &&
-            account.landing_company_short !== 'bvi'
-            ? account.landing_company_short?.charAt(0).toUpperCase() + account.landing_company_short?.slice(1)
-            : account.landing_company_short?.toUpperCase();
+    const getShortCodeAndRegion = (account: TDetailsOfEachMT5Loginid) => {
+        let short_code_and_region;
+        if (!is_demo && !is_eu_user) {
+            short_code_and_region =
+                account.landing_company_short &&
+                account.landing_company_short !== 'svg' &&
+                account.landing_company_short !== 'bvi'
+                    ? account.landing_company_short?.charAt(0).toUpperCase() + account.landing_company_short?.slice(1)
+                    : account.landing_company_short?.toUpperCase();
+            short_code_and_region =
+                account.market_type !== 'financial' && account.landing_company_short !== 'bvi'
+                    ? `${short_code_and_region} - ${account?.server_info?.geolocation?.region}`
+                    : short_code_and_region;
```

Comment:
Why are we defining short_code_and_region 2 times?

Reply:
@matin-binary, I'm not redefining it, i'm reassigning it a value based on another check to add the region to it at the end

Reply:
in that case we need to have 2 separate variables.